devtools/cli/src/main/java/io/quarkus/cli/Build.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.cli;
+
+import java.util.concurrent.Callable;
+
+import io.quarkus.cli.core.ExecuteUtil;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "build", mixinStandardHelpOptions = false, description = "Build your quarkus project")
```

Comment:
looks like I can do `quarkus clean build` and that gets execute as multiple commands. That makes it slower than corresponding `mvn clean build` or `gradle clean build`

Wondering if we could "group" these rather than be done separately ?

Reply:
haven't played with it but picocli lets you customize the execution strategy. https://picocli.info/#_executing_commands_with_subcommands

maybe that could work ?

Reply:
creating a strategy was the trick.

Reply:
I used a strategy.  That worked.